packages/nx-aws-cache/src/tasks-runner/aws-cache.ts

```diff
@@ -170,12 +169,16 @@ export class AwsCache implements RemoteCache {
     }
   }
 
-  private async uploadFile(hash: string, tgzFilePath: string): Promise<void> {
+  private getS3Key(tgzFileName: string) {
+    return join(this.path, tgzFileName);
+  }
+
+  private async uploadFile(hash: string, sourceFile: string | Readable): Promise<void> {
```

Comment:
`sourceFile` type `string` is not needed, you can remove this type and the logic below.

Reply:
Fixed